src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$font: 'Roboto', sans-serif;
+$main-color: #060b35;
+$secondColor: #616070;
+$fontWeightStandard: 400;
+$activeColor: #00acdc;
+$columnsAmount: 1;
```

Comment:
The variable name `$columnsAmount` should also follow the kebab-case naming convention. Consider renaming it to `$columns-amount`.